views/reports/custom_reports.php

```diff
@@ -0,0 +1,162 @@
+                        <label for="race-1">
+                        <input type="checkbox" name="race" id="race-1" value="African American">
+                        African American
+                        </label>
+                    </div>
+                    <div class="checkbox">
+                        <label for="race-2">
+                        <input type="checkbox" name="race" id="race-2" value="Multi Racial">
+                        Multi Racial
+                        </label>
+                    </div>
+                    <div class="checkbox">
+                        <label for="race-3">
+                        <input type="checkbox" name="race" id="race-3" value="Latino">
+                        Latino
+                        </label>
+                    </div>
+                    <div class="checkbox">
+                        <label for="race-4">
+                        <input type="checkbox" name="race" id="race-4" value="Pacific Islander">
+                        Pacific Islander
+                        </label>
+                    </div>
+                    <div class="checkbox">
+                        <label for="race-5">
+                        <input type="checkbox" name="race" id="race-5" value="Native American">
+                        Native American
+                        </label>
+                    </div>
+                    <div class="checkbox">
+                        <label for="race-6">
+                        <input type="checkbox" name="race" id="race-6" value="Other">
+                        Other
+                        </label>
+                    </div>
+                </div>
+            </div>
+            <!-- Multiple Checkboxes -->
+            <div class="form-group row">
+                <label class="col-md-2 col-form-label" for="age"><b>Age</b></label>
+                <div class="col-md-4">
+                    <div class="checkbox">
+                        <label for="age-0">
+                        <input type="checkbox" name="age" id="age-0" value="January">
+                        20-40
+                        </label>
+                    </div>
+                    <div class="checkbox">
+                        <label for="age-1">
+                        <input type="checkbox" name="age" id="age-1" value="February">
+                        41-64
+                        </label>
+                    </div>
+                    <div class="checkbox">
+                        <label for="age-2">
+                        <input type="checkbox" name="age" id="age-2" value="March">
+                        65+
+                        </label>
+                    </div>
+                </div>
+            </div>
+            <!-- Button -->
+            <div class="form-group">
+                <div class="col-md-4">
+                    <a href="custom_reports_table.php" class="btn btn-primary">Generate Report</a>
```

Comment:
Change `custom_reports_table.php` to `/custom-reports-table`. All page navigation is through routes.

Reply:
Also don't use the class `btn-primary`, use `cpca` instead.